### ISSUE TITLE
Score for sites using revoked certs

### ISSUE THREAD

**Issue by anand-bhat** [ssllabs/ssllabs-scan#504]
A few sites I tested ([webcms.axisbank.co.in](https://www.ssllabs.com/ssltest/analyze.html?d=webcms.axisbank.co.in), [caslite.axisbank.co.in](https://www.ssllabs.com/ssltest/analyze.html?d=caslite.axisbank.co.in&ignoreMismatch=on&latest)) score an F as their certificates are revoked. Shouldn't these score a T, rather than an F, as it relates to trust, similar to expired certs or unknown CA?

**Comment by RobTho** [comment]
unknown CA or selfsigned (which get an T) might be good and "safe" even you cannot validate it. 
But if you revoke an cert, you have a good reason to do so,(cause it was compromised), so revoked certs very likely to be unsecure.

**Comment by ivanr** [comment]
Yes, that's the thinking. However, in that light, perhaps expired certificates should also get an F.
